quadlets/foundryvtt/foundryvtt.container

```diff
@@ -0,0 +1,36 @@
+[Unit]
+Description=Foundry VTT Quadlet
+Wants=network-online.target
+After=network-online.target
+After=local-fs.target
+
+[Service]
+Restart=always
+TimeoutStartSec=900
+
+[Install]
+WantedBy=multi-user.target default.target
+
+[Container]
+Image=docker.io/felddy/foundryvtt:release
+ContainerName=foundryvtt
+HostName=foundryvtt
+#AutoUpdate=registry
+
+# Set this instead to expose it locally for asset uploads etc
+Volume=./foundryvtt:/data:z
```

Comment:
Named volumes are located at `~/.local/share/containers/storage/volumes/<volume>/` with a `_data` directory inside. I think named volumes need better documentation and exposure, but you don't need to mount `./foundryvtt:/data`. You can use the named volume location to drop in assets just the same. I am trying to default to named volumes for this repository because `foo.volume` does the `mkdir` for the user. Feedback for a missing path is not great for quadlets (it's in the system journal) and will cause a failure to launch, so forgetting to run `mkdir foundryvtt` can lead to frustration.